### PR TITLE
fix: My gears - top cards uneven height

### LINKED ISSUE #12: bug: [My Gears page] top cards hight
**Describe the bug**
The height of the top card in my gear is not right compared to the others because the height of the cards images is incorrect." 

**To Reproduce**
1. Go to 'My Gear tab'
2 Analyze the top row of cards

**Expected behavior**
The card height has to be the same as the others in the middle row and the bottom row

**Screenshots**
image
the bug ^

image

and want to be^

### PR DESCRIPTION
**Pull request by Sameerdeshmukh18**
## What does this PR do?

in my gears section I limited the height of image to 380px so that the cards wont overgrow if image height is higher than 380px.

Fixes #12 

image
image

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Go to 'My Gear tab'
2 Analyze the size of cards

### CODE REVIEW

File: components/UI/PortfolioItem.jsx [hunk 1]
```diff
@@ -30,8 +30,8 @@ const PortfolioItem = (props) => {
           )}
 
           <div className='bg-transparent'>
-            <div className={`${classes.portfolio__img}`}>
-              <Image alt={title} src={img} width={380} height={1} />
+            <div className={`${classes.portfolio__img}`} >
+              <Image alt={title} src={img} width={380} height={1} style={{maxHeight: "380px"}}/>
```
Comment (piyushgarg-dev) [inline components/UI/PortfolioItem.jsx:34]: Maybe we require some kind of overflow statement here as we are fixing the maxHeight 🙌🏻

**Review by piyushgarg-dev — CHANGES_REQUESTED** [review]

File: components/UI/PortfolioItem.jsx (hunk 1 above)
Comment (Sameerdeshmukh18) [reply components/UI/PortfolioItem.jsx:34]: sorry I didnt understand, could you please elaborate what is expected

Comment (TheMohit2003) [reply components/UI/PortfolioItem.jsx:34]: @Sameerdeshmukh18 ,if the image's height exceeds the specified maximum height of 380 pixels, then the [overflow](https://www.w3schools.com/css/css_overflow.asp) property comes into play. Example :  
` style={{ maxHeight: "380px", overflow: "auto" }}`

Comment (Sameerdeshmukh18) [reply components/UI/PortfolioItem.jsx:34]: yeah ok
I will do that

**Comment by TheMohit2003** [comment]
Hey @Sameerdeshmukh18 , it was all going perfect , why did you close your PR . I know it is confusing sometimes , but you can ask over here . Reopen the PR bro !

**Pull request closed without merging**